pom.xml

```diff
@@ -1840,7 +1840,7 @@ under the License.
 						<configuration>
 							<rules>
 								<requireMavenVersion>
-									<version>[3.8.6]</version>
+									<version>[3.8.6, 3.9.6]</version>
```

Comment:
We'd better do not change the parent pom here, we can add another PR to do this work, because it's not related to this PR, right?

Reply:
It is only for my local build. Should be removed.